src/containers/AdminCampaignStats/index.jsx

```diff
@@ -108,6 +122,22 @@ class AdminCampaignStats extends React.Component {
       disableVanSyncButton: true
     });
 
+  prevCampaignClicked = () => {
```

Comment:
Sorry, our request on #1087 should've been more specific, as we actually find it most useful to have these buttons available on the [`AdminCampaignEdit`](https://github.com/politics-rewired/Spoke/blob/main/src/containers/AdminCampaignEdit/index.jsx) page. I think it's fine to keep on the Campaign Stats page as well, as other users may find it useful here.